data/src/main/java/com/playone/mobile/data/model/IPlayoneEntity.kt

```diff
@@ -0,0 +1,6 @@
+package com.playone.mobile.data.model
+
+/**
+ * The basic interface for that the playone entity can overload.
+ */
+interface IPlayoneEntity
```

Comment:
how about change the name to drop `I`? Because the style is quite old. What do you think?

Reply:
@{edaf591b-a131-4eae-bbf7-56e29eab271c,Lekaha}

Sure, I don't have a good idea. If you have, please let me know 😀

Reply:
I don't have good idea, either. For example, `PlayoneItem`?

Reply:
Sure! Great 🎉